references/segmentation/presets.py

```diff
@@ -1,39 +1,42 @@
 import torch
-import transforms as T
+from torchvision.prototype import features, transforms as T
+from torchvision.prototype.transforms import functional as F
+from transforms import RandomCrop
 
 
-class SegmentationPresetTrain:
-    def __init__(self, *, base_size, crop_size, hflip_prob=0.5, mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225)):
-        min_size = int(0.5 * base_size)
-        max_size = int(2.0 * base_size)
+class WrapIntoFeatures(T.Transform):
+    def forward(self, sample):
+        image, mask = sample
+        return image, features.Mask(F.to_image_tensor(mask).squeeze(0), dtype=torch.int64)
```

Comment:
@pmeier Similarly here, I don't think we should be calling `to_image_tensor` which casts the input to an Image and then rewrap it to a Mask. I'll send a PR with a potential correction, to review.